.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Verify repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@d7b76f909a9ce4f0102c606459aa23439865bd38 # 1.50.0
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
```

Comment:
Ruby is preinstalled: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#language-and-runtime

Do we need a specific version or could we just use the default version?

Reply:
Unfortunately the default version doesn't work. See [this run](https://github.com/ErikSchierboom/website-copy/runs/1383907454?check_suite_focus=true) on my fork.